src/test/php/xml/unittest/NodeTest.class.php

```diff
@@ -184,7 +184,7 @@ public function fromObject() {
       "  <name>Name goes here</name>\n".
       "  <__id/>\n".
       "</node>",
-      $this->sourceOf(Node::fromObject(newinstance(Object::class, [], [
+      $this->sourceOf(Node::fromObject(newinstance('lang.Object', [], [
```

Comment:
`Object::class` should be left untouched. It won't work with older PHP version, but it works w/ Travis...